### PR TITLE
Bump to version 1.2.3

### PR DESCRIPTION
**Pull request by johanjeppsson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* ~~[ ] Bumped the build number (if the version is unchanged)~~
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

### CODE REVIEW

**Review by beckermr — CHANGES_REQUESTED** [review]
CI did not run

**Comment by beckermr** [comment]
@conda-forge-admin restart ci

**Comment by beckermr** [comment]
@conda-forge-admin please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+  sha256: 9ca9e9c8e894468b18e6112b5648d22ffc55cb7819afb16986f0fcac13a3dff7
   patches:
     # https://github.com/scipy/scipy/issues/9777
     # we build with numpy1.9, but test with numpy1.14, so that test still needs to be disabled
     # - np114.patch
 
 build:
-  number: 2
+  number: 0
   skip: true  # [win or aarch64 or ppc64le or py3k]
```
Comment (beckermr) [inline recipe/meta.yaml:17]:
```suggestion
  skip: true  # [win or aarch64 or ppc64le]
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by mattip** [comment]
Outdated, can be closed

**Pull request closed without merging**